$OpenBSD$

Add Data.ByteString.Char8 import needed by patch-http2-client-run.

--- src/Simplex/Messaging/Transport/HTTP2/Client.hs.orig
+++ src/Simplex/Messaging/Transport/HTTP2/Client.hs
@@ -27,6 +27,7 @@
 import Control.Concurrent.Async
 import Control.Exception (IOException, try)
 import qualified Control.Exception as E
+import qualified Data.ByteString.Char8 as B
 import Control.Monad
 import Data.Functor (($>))
 import Data.Time (UTCTime, getCurrentTime)
